tests/test_generators.py

```diff
@@ -9,6 +11,15 @@
 from populous import generators
 from populous.generators.base import NullableMixin
 from populous.generators.base import UniquenessMixin
+from populous.generators.date import to_timestamp
+
+
+@contextmanager
+def not_raises(exception):
+    try:
+        yield
+    except exception:
+        raise pytest.fail("DID RAISE {0}".format(exception))
```

Comment:
found this code when googling for "how to test if an exception is not raised with py.test"